backend/djangoindia/settings/base.py

```diff
@@ -140,4 +140,12 @@
 # https://docs.djangoproject.com/en/4.2/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
-DEFAULT_FROM_EMAIL = "admin@djangoindia.org"
\ No newline at end of file
+DEFAULT_FROM_EMAIL = "admin@djangoindia.org"
+
+
+# CELERY STUFF
+CELERY_BROKER_URL = 'amqp://localhost'  # URL to RabbitMQ server
+CELERY_RESULT_BACKEND = 'rpc://'  # Backend to store task results
+CELERY_ACCEPT_CONTENT = ['json']
+CELERY_TASK_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'json' # default RabbitMQ credentials
```

Comment:
We'll take them from .env , add them in .env.example as well
```suggestion
CELERY_BROKER_URL = os.getenv("CELERY_BROKER_URL")
CELERY_RESULT_BACKEND = os.getenv("CELERY_RESULT_BACKEND")
```

Reply:
Also add `celery` in installed apps.